firebase_admin/exceptions.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 20190 Google Inc.
```

Comment:
I should've known we'd still be on python 3 in the year 20190...

Reply:
Future proof for millenniums to come :rofl:

Reply:
Done